Jenkinsfile

```diff
@@ -47,118 +47,144 @@ properties(
-                    ]
-                )
-                echo "test-pipelines/${params.CUSTOM_PIPELINE} #${customResult.number} succeeded."
-            }
-            if(exception != null) {
-                echo "Custom Pipeline passed, but Regular pipeline failed!"
-                throw exception
+
+                dir(path: 'docs/html') {
+                    sh """# Move the clamav-documentation here.
+                    cp -r ../../clamav_documentation/* .
+                    # Clean-up
+                    rm -rf ../../clamav_documentation
+                    rm -rf .git .nojekyll CNAME Placeholder || true
+                    """
+                }
+
+                dir(path: 'build') {
+                    sh """# CPack
+                    cmake .. -D VENDOR_DEPENDENCIES=ON
+                    cpack --config CPackSourceConfig.cmake """
+                    archiveArtifacts(artifacts: "clamav-${params.VERSION}*.tar.gz", onlyIfSuccessful: true)
+                }
+
+                cleanWs()
             }
         }
 
-        tasks["fuzz_regression"] = {
-            stage("Fuzz Regression") {
-                final fuzzResult = build(job: "test-pipelines/${params.FUZZ_PIPELINE}",
-                    propagate: true,
-                    wait: true,
-                    parameters: [
-                        [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NAME', value: "${JOB_NAME}"],
-                        [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NUMBER', value: "${BUILD_NUMBER}"],
-                        [$class: 'StringParameterValue', name: 'TESTS_FUZZ_BRANCH', value: "${params.TESTS_FUZZ_BRANCH}"],
-                        [$class: 'StringParameterValue', name: 'FUZZ_CORPUS_BRANCH', value: "${params.FUZZ_CORPUS_BRANCH}"],
-                        [$class: 'StringParameterValue', name: 'VERSION', value: "${params.VERSION}"]
-                    ]
-                )
-                echo "test-pipelines/${params.FUZZ_PIPELINE} #${fuzzResult.number} succeeded."
+        stage('Build') {
+            steps {
+                script{
+                    buildResult = build(job: "test-pipelines/${params.BUILD_PIPELINE}",
+                            propagate: true,
+                            wait: true,
+                            parameters: [
+                                    [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NAME', value: "${JOB_NAME}"],
+                                    [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NUMBER', value: "${BUILD_NUMBER}"],
+                                    [$class: 'StringParameterValue', name: 'FRAMEWORK_BRANCH', value: "${params.FRAMEWORK_BRANCH}"],
+                                    [$class: 'StringParameterValue', name: 'VERSION', value: "${params.VERSION}"],
+                                    [$class: 'StringParameterValue', name: 'SHARED_LIB_BRANCH', value: "${params.SHARED_LIB_BRANCH}"]
+                            ]
```

Comment:
Some of the indentation here is pretty deep - I think 8 spaces instead of 4.  I recommend reducing to 4 throughout, for consistency and legitibility.